releasenotes/notes/initialize-ferop-with-tuple-as-label-0dc3f1dbd754ca29.yaml

```
@@ -0,0 +1,14 @@
+---
+features:
+  - |
+    Support the initialization of FermionicOp with tuple as label.
+    :class:`~qiskit_nature.operators.second_quantization.FermionicOp` can be initialized using
+    a tuple of integers, `(action, index)`, like below:
+
+      from qiskit_nature.operators.second_quantization import FermionicOp
+
+      FermionicOp(
+          [([(0, 2), (1, 10)], 2), ([(0, 3)], 34)],
+          register_length=4,
+          display_format="sparse",
+      )
```

```suggestion
      )
      
```
I'm assuming that this blank line is what is needed to make the docs generation happy? :thinking:

---

@ikkoham I guess it didn't help.. Maybe attempting to fix this locally is the best approach then..

---

I think :: is correct (not :).
https://github.com/Qiskit/qiskit-nature/pull/506/commits/0f388758729129b40d9523cdb79309bff08837a5

---

Yep looks good. Now we just need to fix the main CI again :smile: